plugins/services/src/js/stores/__tests__/MesosLogStore-test.js

```diff
@@ -24,23 +27,24 @@ describe('MesosLogStore', function () {
   });
 
   afterEach(function () {
+    MesosLogStore.stopTailing('/bar', true);
     RequestUtil.json = this.requestFn;
   });
 
   describe('#startTailing', function () {
 
     it('should return an instance of LogBuffer', function () {
-      var logBuffer = MesosLogStore.get('/bar');
+      var logBuffer = MesosLogStore.getLogBuffer('/bar');
       expect(logBuffer instanceof LogBuffer).toBeTruthy();
     });
 
   });
 
   describe('#stopTailing', function () {
 
-    it('should return an instance of LogBuffer', function () {
-      MesosLogStore.stopTailing('/bar');
-      var logBuffer = MesosLogStore.get('/bar');
+    it('shouldn\'t return an instance of LogBuffer', function () {
```

Comment:
Please add a test case for `clearData` as we need should verify that it's properly working to not risk memory leaks.